code/controllers/subsystem/non_firing/SSmaprotate.dm

```diff
@@ -73,4 +74,48 @@ SUBSYSTEM_DEF(maprotate)
 	else
 		log_startup_progress("There is no special rotation defined for this day")
 
-
+/datum/controller/subsystem/maprotate/proc/decide_next_map()
+	var/list/potential_maps = list()
+	for(var/x in subtypesof(/datum/map))
+		var/datum/map/M = x
+		if(!initial(M.voteable))
+			continue
+			// And of course, if the current map is the same
+		if(istype(SSmapping.map_datum, M))
+			continue
+			// Or the map from last round
+		if(istype(SSmapping.last_map, M))
+			continue
+		potential_maps[M] = 1
```

Comment:
This can be zero https://discord.com/channels/145533722026967040/727448459862343711/1299526275626696736
```suggestion
		potential_maps[M] = 0
```

Reply:
will look at this one last, think there was another eason for this

Reply:
done